health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/BoundarySelection.js

```
@@ -121,39 +113,41 @@ const BoundarySelection = ({ onSelect, props: customProps, ...props }) => {
+              size={"large"}
+              variation={"secondary"}
+              label={t("MP_ACK")}
+              title={t("MP_ACK")}
+              onClick={() => {
+                setShowPopup(false);
+                //   setCanUpdate(true);
+              }}
+            />,
+          ]}
+          // sortFooterChildren={true}
+        ></PopUp>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)